### PR TITLE
Improved Game picker

### PR DESCRIPTION
**Pull request by flamecoals**
This will be used in the room too, so we can allow people to switch between games.

#### Checklist

* [X] Use a separate branch in your local repo (not `master`).
* [X] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

File: web/src/infra/common/components/alert/AlertLayer.tsx [hunk 1]
```diff
@@ -1,30 +1,26 @@
 import React from 'react';
+import css from './/AlertLayer.css';
```
Comment (jasonharrison) [inline web/src/infra/common/components/alert/AlertLayer.tsx:2]: Two //'s?

File: web/src/infra/common/components/game/GamePickerModal.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+} 
+
+.ButtonContainer {
+  height: 32px;
+  display: flex;
+  padding: 16px;
+}
+
+.Button {
+  margin: '16px 0';
```
Comment (jasonharrison) [inline web/src/infra/common/components/game/GamePickerModal.css:27]: Quotes around margin value not needed

File: web/src/infra/common/components/game/GamesList.tsx [hunk 3]
```diff
@@ -61,10 +42,34 @@ export class GamesList extends React.Component<Props, State> {
...
         </MobileView>
-        <div style={{ margin: '0 4px', display: 'flex', flexWrap: 'wrap' }}>{gamesList}</div>
+        {this.renderGames()}
       </div>
     );
   }
+
+  renderGames() {
+    const { searchQuery } = this.state;
+    let gamesList: JSX.Element[];
```
Comment (jasonharrison) [inline web/src/infra/common/components/game/GamesList.tsx:52]: Use React.ReactNode[]

File: web/src/infra/common/components/game/GamesList.tsx [hunk 4]
```diff
@@ -74,8 +79,29 @@ export class GamesList extends React.Component<Props, State> {
...
+  renderGameCard(game: IGameDef) {
+    if (this.props.gamePickedCallback) {
+      return (
+        <a className={css.Card} onClick={this.onClick(game)} key={game.code}>
+          <GameCard game={game} isLink={true} />
+        </a>
+      );
+    } else {
+      return (
+        <Link href={`/play/[gameCode]`} as={`/play/${game.code}`} key={game.code}>
```
Comment (jasonharrison) [inline web/src/infra/common/components/game/GamesList.tsx:91]: Use passHref prop for Link component https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

File: web/src/infra/lobby/NewRoomModal.css [hunk 5]
```diff
@@ -33,5 +33,5 @@
 }
 
 .Button {
-  margin: '16px 0'
+  margin: '16px 0';
```
Comment (jasonharrison) [inline web/src/infra/lobby/NewRoomModal.css:36]: No quotes needed

**Review by jasonharrison — CHANGES_REQUESTED** [review]

File: web/src/infra/common/components/game/GamesList.tsx (hunk 4 above)
Comment (jasonharrison) [reply web/src/infra/common/components/game/GamesList.tsx:91]: Actually not sure if this is needed

File: web/src/infra/common/components/alert/AlertLayer.tsx (hunk 1 above)
Comment (flamecoals) [reply web/src/infra/common/components/alert/AlertLayer.tsx:2]: Done. Fixed.

File: web/src/infra/common/components/game/GamesList.tsx (hunk 4 above)
Comment (flamecoals) [reply web/src/infra/common/components/game/GamesList.tsx:91]: The child isnt a custom component that wraps an <a, it is an <a...

File: web/src/infra/common/components/game/GamesList.tsx (hunk 3 above)
Comment (flamecoals) [reply web/src/infra/common/components/game/GamesList.tsx]: Done

File: web/src/infra/lobby/NewRoomModal.css (hunk 5 above)
Comment (flamecoals) [reply web/src/infra/lobby/NewRoomModal.css]: Done, thanks for catching.

**Review by jasonharrison — APPROVED** [review]

**Pull request merged**